llvm/lib/ObjCopy/ELF/ELFObject.cpp

```diff
@@ -766,8 +766,14 @@ Error SymbolTableSection::removeSymbols(
     function_ref<bool(const Symbol &)> ToRemove) {
   Symbols.erase(
       std::remove_if(std::begin(Symbols) + 1, std::end(Symbols),
-                     [ToRemove](const SymPtr &Sym) { return ToRemove(*Sym); }),
-      std::end(Symbols));
+               [ToRemove](const SymPtr &Sym) {
+                   if (ToRemove(*Sym)) {
+                       llvm::outs() << "Symbols Removed:" << Sym->Name<< "\n";
```

Comment:
This also needs to be guarded by an if (Verbose). You can add the isVerboseFlag to SymbolTableSection as well and set it through `Obj.SymbolTable->isVerboseFlag = ...`